addons/website_sale_delivery/static/src/js/website_sale_delivery.js

```diff
@@ -14,6 +16,17 @@ odoo.define('website_sale_delivery.checkout', function (require) {
         var $amount_total = $('#order_total span.oe_currency_value');
         var $carrier_badge = $('#delivery_carrier input[name="delivery_type"][value=' + result.carrier_id + '] ~ .badge.hidden');
         var $compute_badge = $('#delivery_carrier input[name="delivery_type"][value=' + result.carrier_id + '] ~ .o_delivery_compute');
+        var $discount = $('#order_discounted');
+
+        if ($discount && result.new_amount_order_discounted) {
+            // Cross module without bridge
+            // Update discount of the order
+            $discount.find('.oe_currency_value').text(result.new_amount_order_discounted);
+
+            // We are in freeshipping, so every carrier is Free
+            $('#delivery_carrier .badge').text(_t('Free'));
```

Comment:
Export translation

Reply:
@nim-odoo 
translation exported,
please approve #enterprise/2651 at the same time